### ISSUE TITLE
ESP8266 documentation for repl connection

### ISSUE THREAD

**Issue by blakev** [micropython/micropython#2840]
In the [documentation for the repl](https://github.com/micropython/micropython/blob/master/docs/esp8266/tutorial/repl.rst) it has this line for connecting via linux:

```
$ picocom /dev/ttyUSB0
```

However, this is insufficient to get a true interactive session. Instead it should be:

```
$ picocom /dev/ttyUSB0 --b 115200 --omap crcrlf --imap crcrlf
```

**Comment by kfricke** [comment]
What do I miss without an additional line feed?

**Comment by blakev** [comment]
Ahh you're correct @kfricke ~ however, if the baudrate isn't set it appears like it's not working.

```
$ picocom /dev/ttyUSB0 --b 115200
```

**Comment by dpgeorge** [comment]
Thanks for the tip, fixed in 3217bbe4910416b9084691ea5ff1ec1530e15ee4.
